app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+
+  validates :name, uniqueness: true, presence: true
+  validates_presence_of :password    # What's the difference? :   validates :email, presence: true
+  validates_presence_of :email
+# Do I need these? 
```

Comment:
Nope. Some websites do that to make sure your email is correct, but for our purposes it isn't necessary.